_posts/2021-03-01-changelog-142.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #142"
+date: 2021-03-01 17:00:00 +0300
+---
+
+
+## New Features
+
+* Debugging is now available in [PyCharm Professional](https://www.jetbrains.com/pycharm/) starting with 2021.1 EAPs
```

Comment:
Let's add a note, that it requires [Native Debugging Support](https://plugins.jetbrains.com/plugin/12775-native-debugging-support) plugin and only LLDB is supported